src/components/pages/Landing/LandingSectionOne.js

```diff
@@ -7,6 +8,11 @@ export default class LandingSectionOne extends Component {
   }
 
   render(){
-    return <h1>Stub for landing section two</h1>
+    return (
+      <div>
+        <h1>Stub for landing section two</h1>
+        <InitiativeCard img="/images/sevatiffin.jpeg" title="Seva Tiffin" description="Food is served at the several locations for the needy at Hospitals, Schools for underprivileged children, Old age Homes. If you wish to serve at any of these locations you can Sign Up for Seva. You can also buy a meal for the location which will be delivered to the needy by Volunteers." />
```

Comment:
Like we previously did for "LandingSectionTwo" part where the props data was actually taken from other file, we can do the same here. The props data insted of hardcoding it directly in the component we can import it

Reply:
Yeah, we would do that in this task - https://www.proso.io/tasks/5cbb316e76374c5262ecdb1f

This task involved just creating one child of `InitiativeGroup` component. I just needed a way to demo it. We can remove it once the `InitiativeGroup` component is created.